src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -194,6 +195,12 @@ export const exchangeStitchingEnvironment = ({
     extend type Me {
       orders(first: Int, last: Int, after: String, before: String, mode: CommerceOrderModeEnum, sellerId: String, sort: CommerceOrderConnectionSortEnum, states: [CommerceOrderStateEnum!]): CommerceOrderConnectionWithTotalCount
     }
+
+    extend type Mutation {
+      createInquiryOffer(
+        input: CommerceCreateInquiryOfferOrderWithArtworkInput!
+      ): CommerceCreateInquiryOfferOrderWithArtworkPayload
```

Comment:
Note here that we are still referencing the underlying exchange mutation types, but with a new name for the Mutation rather than monkey-patching.

Reply:
Ah, so that's why you were doing this in the stitching file (vs. a regular old MP mutation).